cmd/agent/app/processors/thrift_processor_test.go

```diff
@@ -121,7 +121,7 @@ type failingHandler struct {
 	err error
 }
 
-func (h failingHandler) Process(_ context.Context, iprot, oprot thrift.TProtocol) (success bool, err thrift.TException) {
+func (h failingHandler) Process(context.Context, thrift.TProtocol, thrift.TProtocol) (success bool, err thrift.TException) {
```

Comment:
I don't think this is a great approach. It's ok in some cases to replace name with _ or even drop it when the type is reasonably unique, like Context or zap.Logger. But when it's either very common types like `string` or repeated type as in this case, a better approach is change the name into a comment, e.g.
```suggestion
func (h failingHandler) Process(context.Context, /* in */ thrift.TProtocol, /* out */ thrift.TProtocol) (success bool, err thrift.TException) {
```

Reply:
Understood. I will make the changes.